packages/bridge/contracts/l1/L1DaiGateway.sol

```diff
@@ -81,51 +102,102 @@ contract L1DaiGateway is L1ArbitrumExtendedGateway {
     emit Closed();
   }
 
-  function createOutboundTx(
-    address _from,
-    uint256 _tokenAmount,
-    uint256 _maxGas,
-    uint256 _gasPriceBid,
-    uint256 _maxSubmissionCost,
-    bytes memory _outboundCalldata
-  ) internal override returns (uint256) {
+  function outboundTransfer(
+    address l1Token,
+    address to,
+    uint256 amount,
+    uint256 maxGas,
+    uint256 gasPriceBid,
+    bytes calldata data
+  ) external payable returns (bytes memory) {
     // do not allow initiating new xchain messages if bridge is closed
```

Comment:
Difference from the official Arbitrum code `require(isRouter(msg.sender), "NOT_FROM_ROUTER");`. `outboundTransfer` is not permissionlessly callable in the official implementation. Not saying this is necessarily a problem, but we need to keep it in mind evaluating the security differences.

Reply:
It's fine AFAIK.

Reply:
Ok, I'm not sure how to verify this as I don't know why Arbitrum team added it in the first place.

Reply:
For a single asset gateway this shouldn't be an issue